maintainers/scripts/README.md

```diff
@@ -0,0 +1,22 @@
+# Maintainer scripts
+
+This folder contains various executable scripts for nixpkgs maintainers,
+and supporting data or nixlang files as needed.
+
+What follows is a (very incomplete) overview of available scripts.
+
+
+## Metadata
+
+### `get-maintainer.sh`
```

Comment:
Something this is missing is the motivation for why we need this. Can you describe the use case for this script here?

Reply:
Still good to write this down, but I just had a need for this script myself! Had to find out what the GitHub handle of somebody on Matrix was :D

Reply:
Ah yeah you're right, but yes that text would be great to add!